packages/client/webpack.config.js

```diff
@@ -1,8 +1,17 @@
 const { resolve } = require('path')
+const { ProvidePlugin } = require('webpack')
 
 module.exports = {
   mode: 'production',
   entry: './dist.browser/browser/index.js',
+  plugins: [
+    new ProvidePlugin({
+      Buffer: ['buffer', 'Buffer'],
+    }),
+    new ProvidePlugin({
+      process: 'process/browser',
+    }),
+  ],
```

Comment:
thanks, and I added `process/browser` because one node_module in particular was using `process.env.SOME_VARIABLE` and adding this seemed to be the best way to harden the bundle from breaking from any other module checking `process.env` in the future.

Reply:
I assume there aren't any other gotcha node types you can think of that would break in the browser version of the client thanks to webpack v5 not polyfilling in obscure edge cases, right?

Reply:
so far i believe i've added everything we need, webpack 5 is quite vocal if it is missing a module it needs as part of the bundle, so in case something gets added in the future we should see the error and can add it appropriately. 

although it would be nice if we had some kind of ci run that actually used the webpack bundle, it would have likely discovered this "Buffer is undefined" error before it was reported to us by a user. lately i am thinking this would be much more useful (or at least, in addition to) the karma-typescript setup we have for the client specifically.